src/components/Table/TableDetailWizard/TableDetailWizardHeader/TableDetailWizardHeader.jsx

```diff
@@ -20,8 +21,8 @@ const StyledButton = styled.div`
   margin-left: auto;
 
   .bx--btn__icon {
-    margin-left: 5px;
-    margin-right: 5px;
+    margin-left: ${spacing02};
```

Comment:
verify that layout does not change.

Reply:
This class is no longer used it appears. The close button receives classes of `bx--btn bx--btn--ghost`